Form.css

```diff
@@ -0,0 +1,24 @@
+.Form {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.Form * {
+  margin-bottom: 30px;
+}
+
+.Form input {
```

Comment:
Ideally, elements should be styled with classes rather than their identity. This allows multiple styles to be "composed" on to an element, making modification easier.

Styling elements with classes also lowers the risk of styles leaking out to unintended children. Another way to be more explicit might be to use the child combinator (`>`). This would ensure that only direct children would receive these styles.

However, in some CSS frameworks (such as BEM) using document structure to define element identity is considered a mistake because moving the element in the DOM would break its styles. Instead of using document relationship, each element would have a unique class selector, something like `.form__input {`.